packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.tsx

```diff
@@ -41,7 +42,7 @@ export class AboutModal extends React.Component<AboutModalProps, ModalState> {
   ariaLabelledBy = `pf-about-modal-title-${this.id}`;
   ariaDescribedBy = `pf-about-modal-content-${this.id}`;
 
-  static defaultProps = {
+  static defaultProps: PickOptional<AboutModalProps> = {
```

Comment:
Why can't we do `static defaultProps = { ... } as AboutModalProps`?

Reply:
You probably can, but if one of the types was wrong, for example if you defaulted isOpen to a string `isOpen: 'false'` the TS error/warning would not be as clear